### DISCUSSION TITLE
Retry test suite when a test fails

### DISCUSSION THREAD

**Discussion by abaracedo** [cypress-io/cypress#17784] [Questions and Help]
Is there a way to retry a test suite when one of its tests fails? 

I have a test suite running the `before` hook to log in a user and preserve the cookies. The thing is that when one of the tests fails when retrying it, it does not have the cookies and all the tests fails because they expect to have a user logged in.

**Comment by christianinsert** [comment]
You might want use [cypress-repeat]( https://www.npmjs.com/package/cypress-repeat) with the ` --rerun-failed-only` option
